examples/mlpod/README.md

```diff
@@ -0,0 +1,21 @@
+### Compile LAMMPS/POD 
+
+Go to `lammps` directory and build with the POD package:
+
+    cd path/to/lammps
+    mkdir build-pod
+    cd build-pod
+    cmake -C ../cmake/presets/basic.cmake -D PKG_ML-POD=on ../cmake
+    cmake --build .
+
+### Fit a POD potential for tantalum
+
+Go to `lammps/examples/pod/Ta` directory and run 
```

Comment:
This file will need to be edited to reflect final agreed path to Ta dir.

Reply:
examples/mlpod/README.md includes a link to another Github repository that hosts a number of other examples for fitting and running POD. These examples are not included in ML-POD because the training data is considerably bigger than the Ta data. There is a separate README.md in the Ta dir for instructions on how to fit POD and run MD.

Reply:
Yes, we'll do that when we agree on the example path.